apps/api/src/app/workflows-v2/usecases/build-step-issues/build-step-issues.usecase.ts

```diff
@@ -173,7 +182,7 @@ export class BuildStepIssuesUsecase {
 
             return acc;
           },
-          {} as Record<string, ContentIssue[]>
+          {} as Record<string, StepContentIssue[]>
```

Comment:
I suggest renaming `StepContentIssue` to `StepIssue`. It's not about content anymore.

Reply:
it is fine in this context, we have StepContentIssue and StepIntegrationIssue that together are StepIssue.
in this function we build StepContentIssue